raiden_contracts/deploy/__main__.py

```diff
@@ -20,7 +20,7 @@
 from raiden_contracts.deploy.contract_verifier import ContractVerifier
```

Comment:
Maybe you want to rename the entities that deploy, manage verify, etc. the contract**s**.

Reply:
The future of these classes are a bit uncertain.  At least there is a wish that ContractVerifier be able to check each contract.  https://github.com/raiden-network/raiden-contracts/issues/1086